sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql

```diff
@@ -801,6 +803,24 @@ aggregates AS (
     SUM(
       scalar_parent_navigator_storage_persist_count
     ) AS scalar_parent_navigator_storage_persist_count_sum,
+    mozfun.stats.mode_last(
+      ARRAY_AGG(scalar_parent_os_environment_is_taskbar_pinned ORDER BY submission_timestamp)
+    ) AS scalar_parent_os_environment_is_taskbar_pinned,
+    COUNTIF(
+      'Desktop' = scalar_parent_os_environment_launch_method
+    ) > 0 AS scalar_parent_os_environment_launched_via_desktop,
+    COUNTIF(
+      'StartMenu' = scalar_parent_os_environment_launch_method
+    ) > 0 AS scalar_parent_os_environment_launched_via_startmenu,
```

Comment:
Since these aren't automated, might be nice to use friendly names:
```suggestion
    ) > 0 AS scalar_parent_os_environment_launched_via_start_menu,
```

Reply:
same for other_shortcut